.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
I think this was already included in https://github.com/xFAANG/askql/pull/525

Reply:
True, true, I will rebase with master since it is already merged.